server/channels/app/export.go

```diff
@@ -113,6 +113,14 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 		return err
 	}
 
+	ctx.Logger().Info("Bulk export: exporting bots")
+	var botPPs []string
```

Comment:
I don't think there is a need to declare `botPPs` here. The `err` variable will be always new, if the first variable is new.